### ISSUE TITLE
class weight in multi output does not work

### ISSUE THREAD

**Issue by sun-peach** [fchollet/keras#6826]
Hi,
I have two outputs in the model. I use Activation(softmax,name="eid_class") and Activation(softmax,name="sid_class") to define them.
During training, I set the class weight dictionary as following:
{"eid_class":{0:1.0, 1:1.6, 2:3.0}}

However, I find the training does not change when I apply this weight compared with the one without weight.

Is there any mistake in my program? Thank you.
